Project.toml

```diff
@@ -0,0 +1,31 @@
+name = "VegaLite"
```

Comment:
I'd prefer not to have a `Project.toml` here for now. What I have seen happening is that the CI tests use this one here, and then folks forget to update the `REQUIRE` file, and then one can actually end up with releasing a version and it passes all CI that doesn't work on end user systems. Once the whole attobot infrastructure is moved over to `Project.toml`, we can obviously add this, so would be great if we can save this somehow!